libfreerdp/core/input.c

```diff
@@ -479,6 +539,9 @@ BOOL input_recv(rdpInput* input, wStream* s)
 {
 	UINT16 i, numberEvents;
 
+	if (!input || !s)
```

Comment:
does not hurt either and the symbol is exported (at least libfreerdp internally)